scripts/build_demand_profiles.py

```diff
@@ -181,8 +181,21 @@ def build_demand_profiles(
     gegis_load = gegis_load.to_dataframe().reset_index().set_index("time")
     # filter load for analysed countries
     gegis_load = gegis_load.loc[gegis_load.region_code.isin(countries)]
-    logger.info(f"Load data scaled with scaling factor {scale}.")
-    gegis_load["Electricity demand"] *= scale
+
+    if isinstance(scale, dict):
+        logger.info(f"Using custom scaling factor for load data.")
+        for country in countries: 
+            scale.setdefault(country,1.0) 
```

Comment:
Great @virio-andreyana :D
I think this PR is basically ready to merge; however, what do you think about whether having this "1.0" fixed,
we have it by config, like:
```suggestion
        logger.info(f"Using custom scaling factor for load data.")
        DEFAULT_VAL = scale.get("DEFAULT", 1.0)
        for country in countries: 
            scale.setdefault(country, DEFAULT_VAL) 
```
Basically the proposal is to optionally treat the country key "DEFAULT" as the default value, if present